### PR TITLE
adds list of default rollup configs

### PR DESCRIPTION
**Pull request by shalper**
# What changed

----

## Pages to review

- https://deploy-preview-583--logz-docs.netlify.app/user-guide/infrastructure-monitoring/data-rollups.html

### CODE REVIEW

File: _source/user-guide/infrastructure-monitoring/rollups.md [hunk 1]
```diff
@@ -9,34 +9,87 @@ contributors:
...
+
+Configuring the rollups for your custom metrics is included in your package and we're happy to offer it!
+
+
+To kick off this process, <a href="mailto:help@logz.io?subject=Requesting a custom metrics rollup config &body= Hi! I'd like to send custom metrics. Here's my info: 1. Logz.io Metrics account ID or token. 2. 5 sample JSONs 3. Dimensions associated with each Metricset.">email Support</a> to request a custom rollups config.
+
+Include the following details in your message:
+
+1. Your Logz.io Metrics account ID or [token](/user-guide/accounts/finding-your-metrics-account-token/).
+2. At least 5 sample JSONs of your custom metrics.
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/rollups.md:91]: This is a little rough, how about 2-3 examples?

File: _source/user-guide/infrastructure-monitoring/rollups.md [hunk 2]
```diff
@@ -9,34 +9,87 @@ contributors:
...
+* System
+* vSphere
+* Windows
+
+### Requesting a custom rollups config
+
+Configuring the rollups for your custom metrics is included in your package and we're happy to offer it!
+
+
+To kick off this process, <a href="mailto:help@logz.io?subject=Requesting a custom metrics rollup config &body= Hi! I'd like to send custom metrics. Here's my info: 1. Logz.io Metrics account ID or token. 2. 5 sample JSONs 3. Dimensions associated with each Metricset.">email Support</a> to request a custom rollups config.
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/rollups.md:86]: Can't they also do it via chat?

File: _source/user-guide/infrastructure-monitoring/rollups.md [hunk 3]
```diff
@@ -9,34 +9,87 @@ contributors:
...
+
+Include the following details in your message:
+
+1. Your Logz.io Metrics account ID or [token](/user-guide/accounts/finding-your-metrics-account-token/).
+2. At least 5 sample JSONs of your custom metrics.
+3. If you are sending multiple metricsets,
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/rollups.md:92]: Not sure I got this sentence..

File: _source/user-guide/infrastructure-monitoring/rollups.md [hunk 4]
```diff
@@ -9,34 +9,87 @@ contributors:
...
 
-When you're investigating incidents, the granularity of the data will vary depending on how much time has passed. As long as you're within 3 days of the incident, you can drill down on the raw data points. As you move past the 3 day mark, you'll be able to drill down on the data at a granularity of 1 minute intervals. Once a week has passed, you'll be able to drill down to a maximum resolution of 10 minute intervals. Once a month or more has passed, the data is at its maximum compaction, and you can drill down on the hourly metrics. 
+As long as you're within 3 days of the incident, you can drill down on the raw data points. As you move past the 3 day mark, you'll be able to drill down on the data at a granularity of 1 minute intervals. Once a week has passed, you'll be able to drill down to a maximum resolution of 10 minute intervals. Once a month or more has passed, the data is at its maximum compaction, and you can drill down on the hourly metrics.
 
-### Available data points
+### Default rollups
+
+The following is a list of rollup configs that are pre-configured and available by default:
+
+* Aws (Major services and namespaces, including EC2, S3, RDS, SQS)
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/rollups.md:62]: We'll probably need to specify exactly which servers are available both on AWS and Azure

File: _source/user-guide/infrastructure-monitoring/rollups.md (hunk 2 above)
Comment (shalper) [reply _source/user-guide/infrastructure-monitoring/rollups.md:86]: Josh asked that they email it. That's a good question for Josh.

File: _source/user-guide/infrastructure-monitoring/rollups.md (hunk 1 above)
Comment (shalper) [reply _source/user-guide/infrastructure-monitoring/rollups.md:91]: Josh...

File: _source/user-guide/infrastructure-monitoring/rollups.md [hunk 5]
```diff
@@ -9,34 +9,87 @@ contributors:
...
+* Azure (Major Azure sevices)
```
Comment (shalper) [inline _source/user-guide/infrastructure-monitoring/rollups.md:63]:
```suggestion
* Azure services
```

File: _source/user-guide/infrastructure-monitoring/rollups.md (hunk 4 above)
Comment (shalper) [inline _source/user-guide/infrastructure-monitoring/rollups.md:62]:
```suggestion
* AWS services and namespaces
```

**Pull request merged**